model/config.go

```diff
@@ -424,6 +429,11 @@ type LdapSettings struct {
 
 	// Customization
 	LoginFieldName *string
+
+	// Login button colors
+	LoginButtonColor       *string
+	LoginButtonBorderColor *string
```

Comment:
Missing defaults. You need to add defaults in the SetDefaults function. (Which has changed into multiple functions so you need to rebase)